tests/test_slack.py

```diff
@@ -0,0 +1,34 @@
+import os
+import json
+from sys import version_info
+
+from mock import mock_open, patch
+import pytest
+import requests_mock
+import requests
+
+from freepacktbook.slack import SlackNotification
+
+def test_should_notify():
+    channel = '#abc'
+    slackUrl = 'http://my.slack.com'
+    slackNotif = SlackNotification(slackUrl, channel)	
+    data= {'title': 'ttt', 'url': 'someUrl.com', 'image_url': 'aa.com', 'book_url': 'bu', 'description': 'desc'}
+
+    with requests_mock.mock() as m:
+	m.post(slackUrl)
```

Comment:
Please use 4 spaces instead of 2.
```suggestion
	  m.post(slackUrl)
```